
geometry_msgs/Pose pose
---
float64[] SO3
